docs/tsc/meetings/2020-05-04.md

```diff
@@ -0,0 +1,128 @@
+<!-- SPDX-License-Identifier: CC-BY-4.0 -->
+<!-- Copyright Contributors to the OpenColorIO Project. -->
+
+May 4, 2020
+
+Host: Michael Dolan
+
+Rotating Secretary: Michael Dolan
+
+Attendees:
+  * [X] Mark Boorer (_TSC_) - Industrial Light & Magic
+  * [X] Sean Cooper (_TSC_) - DNEG
+  * [X] Michael Dolan (_TSC Chair_) - Epic Games
+  * [X] Mei Chu (_TSC_) - Sony Pictures Imageworks
+  * [X] Patrick Hodoul (_TSC_) - Autodesk
+  * [ ] John Mertic - Academy Software Foundation / Linux Foundation
+  * [X] Carol Payne (_TSC_) - Netflix
+  * [ ] Carl Rand (_TSC_) - Weta Digital
+  * [X] Doug Walker (_TSC Chief Architect_) - Autodesk
+  * [X] Kevin Wheatley (_TSC_) - Framestore
+  * [X] Bernard Lefebvre - Autodesk
+  * [X] Joseph Goldstone - ARRI
+  * [X] Troy Sobotka
+
+# **OCIO TSC Meeting Notes**
+
+* GPU CI update
+    - GH Actions is now in place, and LF release engineering is working on 
+      setting up needed AWS infrastructure to provision GPU instances. Once 
+      this is completed (within ~1-2 weeks) we will work with JF to implement
+      into CI. A CI working group goal is for OCIO to have GPU CI up and running
+      by the May CI working group meeting in a month.
+
+* OpenColorIO-configs update
+    - There was a joint ACES-OCIO meeting last Friday to discuss the way 
+      forward for the ACES OCIO config. We will be starting a new ASWF repo to
+      house a new config, and further discuss the design within that repo's 
+      issues.
+    - Doug: Rod Bogart has some action items around licensing of existing code 
+      and HP's python scripts. TBD how to move forward with these.
+    - Joseph: Did anyone from Canon come to this group? The current IDTs dont 
+      match Canon's white paper. It contains a note saying the formula has been 
+      revised.
+    - Doug: That's more on the ACES side to get that updated. It should be 
+      mentioned to Alex and Scott. We're just going to follow what is in their 
+      repo.
+    - Joseph: I will reach out to them via email.
+    - Sean: The tricky part will be what to do with old stuff. Also the 
+      particulars of the config. But path forward is clear. It would be 
+      worthwhile to investigate a chair to oversee config maintenance and to 
+      have a clear owner - whoever has the time and detail.
+    - Mark: That person could also own the numbers in the config, which 
+      sometimes are debated.
+    - Doug: From ACES, Thomas Mansencal and Michael Parsons were identified as 
+      owners. ACES is encouraging OCIO to take ownership and become a product 
```

Comment:
Please replace the sentence "ACES is encouraging ..." with "I think ACES would like OCIO to take a bigger role in the creation and maintenance of the config, similar to a product partner."

Just wanted to clarify that I was stating my interpretation of the comments made at the ACES meeting and it is not an official request from the Academy to OCIO.

Reply:
Also, I hope Thomas and Michael will continue their efforts on the config.  It is greatly appreciated!

Reply:
@KelSolaar , please take note of my comments above, which I hope more accurately reflects what was said at the Monday meeting.  As was said at the Friday meeting, I think everyone would like you to continue the extremely valuable role you have been playing.  The Monday meeting was just trying to recap that for the OCIO TSC.  Just wanted to make sure there was no confusion.  ;)

Reply:
Thanks @doug-walker, time permitting I would be keen to continue indeed and timezone maybe participate in some of the TSC meetings.

Reply:
Fixed